tests/shared/src/test/scala/org/http4s/UrlFormSpec.scala

```diff
@@ -160,12 +158,8 @@ class UrlFormSpec extends Http4sSuite {
     ) {
       Prop.forAll { (map: Map[String, NonEmptyList[String]]) =>
         // non-empty because the kv-constructor can't represent valueless fields
-        val flattened = for {
-          kv <- Chain.fromSeq(map.toSeq)
-          k = kv._1
-          vs = kv._2
-          v <- Chain.fromSeq(vs.toList)
-        } yield k -> v
+        val flattened =
+          Chain.fromSeq(map.toList.flatMap(x => x._2.toList.fproductLeft(_ => x._1)))
```

Comment:
This may have been more readable before 😂

Reply:
C'mon, who will read this (apart from CI)! 😆